src/functionalTest/java/uk/gov/hmcts/reform/professionalapi/OrganisationRetrieveTest.java

```diff
@@ -64,10 +64,14 @@ public void can_retrieve_Pending_and_Active_organisations() {
         assertThat(orgIdentifierTwo).isNotEmpty();
 
         professionalApiClient.updateOrganisation(orgIdentifierTwo, hmctsAdmin);
+        Map<String, Object> newOrgResponse = professionalApiClient.retrieveOrganisationDetails(orgIdentifierTwo,hmctsAdmin);
         Map<String, Object> finalResponse = professionalApiClient.retrieveAllOrganisations(hmctsAdmin);
 
         assertThat(finalResponse.get("organisations")).isNotNull();
         Assertions.assertThat(finalResponse.size()).isGreaterThanOrEqualTo(1);
+        assertThat(newOrgResponse.get("paymentAccount")).asList().size().isEqualTo(3);
+        assertThat(newOrgResponse.get("contactInformation")).asList().size().isEqualTo(2);
+
     }
```

Comment:
more than just checking size, can you  also check the values in the list?

Reply:
i can check the values for contact info but cannot for payment as i dont know what they will be. its a functional test so the pbas are randomised each time the test is ran so they dont cause any conflicts in the DB i presume?